common/respondent_utilities.py

```diff
@@ -80,3 +91,36 @@ def register_respondent(survey_id, period, username, ru_ref=None, verified=True)
     logger.debug('Successfully registered respondent', survey_id=survey_id, period=period,
                  ru_ref=ru_ref, respondent_id=respondent_id)
     return respondent_id
+
+
+def create_phone_number():
+    return format(randint(TELEPHONE_NUMBER_START, TELEPHONE_NUMBER_END), '011d')
+
+
+def create_enrolled_respondent_for_the_test_survey(context):
+    create_respondent_data(context)
+
+    create_respondent(user_name=context.respondent_email, enrolment_code=context.iac, phone_number=context.phone_number)
+    create_respondent_user_login_account(context.respondent_email)
+
+
+def create_unenrolled_respondent(context):
+    create_respondent_data(context)
+    create_respondent(user_name=context.respondent_email, enrolment_code=context.iac, phone_number=context.phone_number)
+    create_respondent_user_login_account(context.respondent_email)
+
+    unenrol_respondent_in_survey(context.survey_id)
```

Comment:
still unenrolling after, is this intentional?

Reply:
Yes, this PR already got bigger than I was intended and it does work as it is.